packages/fxa-content-server/app/styles/modules/_custom-rows.scss

```diff
@@ -18,6 +18,11 @@
     }
   }
 
+  .spaced {
+    margin-left: 10px;
+    margin-right: 10px;
```

Comment:
I think we can add these styles to `.marketing-link` without a new class, do you think we'll use this maybe in the future?

Reply:
That works for me!